src/components/button/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC, MouseEvent, ReactNode } from 'react';
+import { Button as ChakraButton } from '@chakra-ui/react';
+
+interface Props {
+  variant?: 'primary';
+  size?: 'md' | 'lg' | 'xl';
```

Comment:
Where does that spec come from it looks like there is only one size in the Figma file?
https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/AS24-WebComponents

Reply:
You can see the size referencing from [Token figma](https://www.figma.com/file/WmNmJ7jdB0Z7tOjj516nFB/AS24-Tokens?node-id=25%3A222)

Reply:
thanks 🙏